.travis.yml

```diff
@@ -45,10 +45,14 @@ cache: apt
 
 before_install:
  - sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
+ - sudo add-apt-repository "deb http://llvm.org/apt/precise/ llvm-toolchain-precise-3.5 main" -y
```

Comment:
I'm not sure you can add random non ppa repos. Just nick the relevant config lines from ola: https://github.com/OpenLightingProject/ola/blob/master/.travis.yml#L48